pkg/machines/libvirtUtils.js

```diff
@@ -53,7 +53,15 @@ export function prepareDisplaysParam(displays) {
 export function prepareDisksParam(disks) {
     const isVolume = (disk) => disk.source.volume && disk.source.pool;
     const getVolume = (disk) => isVolume(disk) ? `${disk.source.pool}/${disk.source.volume}` : null;
-    const getPath = (disk) => isVolume(disk) ? null : disk.source.file;
+    const getPath = (disk) => {
+        // see: https://libvirt.org/formatdomain.html#elementsDisks -> source
+        switch (disk.type) {
+        case "file": return disk.source.file;
+        case "block": return disk.source.dev;
```

Comment:
Just for clarification, the only way that the code might end up here, is that the default pool will be on non local fs, right?

Reply:
I don't know if I understand question. Why default pool? Vm can have disks from other pools than default right?
Also you can take any part of local fs, create a volume out of that, and then tell VM to see it as block disk, now?

Reply:
Sorry, I was consused because I though that since we pass to virt-install --vol option it will create disk of type volume, which is not true as I verified just now.